src/components/blipDatepicker/index.js

```diff
@@ -257,8 +255,7 @@ export class BlipDatepicker extends Component {
 
     const timeInputContainer = html`
     <div class="${BlipDatepicker.style.timeInputContainer}">
-      <div class="${BlipDatepicker.style.clockIcon}">${raw(Clock)}</div>
```

Comment:
Don't we have the clock Icon on the left side anymore?

Reply:
No. As talked to Anny, we decided to remove the first clock (blue) and keep only the right side clock that receive the click to open the time picker.